ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/response/CCDPaymentIntention.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.hmcts.cmc.ccd.domain.response;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import lombok.Builder;
+import lombok.Value;
+import uk.gov.hmcts.cmc.ccd.domain.CCDPaymentOption;
+import uk.gov.hmcts.cmc.ccd.domain.CCDPaymentSchedule;
+
+import java.math.BigDecimal;
+import java.time.LocalDate;
+
+@Builder
+@Value
+public class CCDPaymentIntention {
+    private CCDPaymentOption paymentOption;
+    private LocalDate paymentDate;
+    private BigDecimal instalmentAmount;
+    private LocalDate firstPaymentDate;
+    private CCDPaymentSchedule paymentSchedule;
+    private LocalDate completionDate;
+    private String paymentLength;
+
+    @JsonIgnore
+    public boolean hasRepaymentPlanInfo() {
+        return null == instalmentAmount
+            && null == firstPaymentDate
+            && null == paymentSchedule
+            && null == completionDate
+            && null == paymentLength;
```

Comment:
paymentLength is string, is just null check enough

Reply:
prefer to use `isBlank` for type `String`

Reply:
added